src/llvm/analysis/old/llvm-debug.h

```diff
@@ -16,7 +16,7 @@
 
 #else
 
-#define DBG(...) ;
+#define DBG(...) do { } while (0)
```

Comment:
I hope this macro will be removed completely in the future. Anyway, this is good for now, thanks.